extensions/api/v1alpha1/sandboxtemplate_types.go

```diff
@@ -31,13 +31,36 @@ limitations under the License.
 package v1alpha1
 
 import (
+	networkingv1 "k8s.io/api/networking/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	sandboxv1alpha1 "sigs.k8s.io/agent-sandbox/api/v1alpha1"
 )
 
 // NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
 // Important: Run "make" to regenerate code after modifying this file
 
+// NetworkPolicySpec defines the desired state of the NetworkPolicy.
+type NetworkPolicySpec struct {
+	Enabled bool `json:"enabled,omitempty"`
+
+	// Keep your high-level helpers (This is "opinionated" value-add)
+	IngressControllerSelectors *IngressSelector `json:"ingressControllerSelectors,omitempty"`
```

Comment:
The name is overly specific. Users might want to whitelist other trusted sources that are not Ingress Controllers. Suggest something more generic, such as TrustedSelectors or AllowedPeers.

Reply:
Will remove, thanks!